app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -83,7 +85,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
         if (Utils.isOnline(getActivity())) {
             setIsLoading(true);
             App.getInstance().getModelCache()
-                .getAsync(Const.CACHE_KEY_NEWS + plusId, new ModelCache.CacheListener() {
+                .getAsync(CACHE_KEY_NEWS + plusId, new ModelCache.CacheListener() {
```

Comment:
Hmmm, I can revert that commit. I did cache keys part in 1 commit. 

Another way to prevent clash of keys is to put the name of the feature in front of the keys. What do you think?